docs/3.x/tracking-javascript-guide.md

```diff
@@ -148,11 +148,13 @@ It is possible to configure Piwik so that it accurately measures the time spent
 _paq.push(['enableHeartBeatTimer']);
 ```
 
-Piwik will then send requests to count the actual time spent on the page, when the user is actively viewing the page (ie. when the tab is active and in focus). These heartbeat requests will not track additional actions or pageviews. By default, Piwik will send a heartbeat request every `15` seconds. You may change the default interval, for example to send a request every `30` seconds:
+Piwik will then send requests to count the actual time spent on the page, when the user is actively viewing the page (i.e. when the tab is active and in focus). The heart beat request is executed when:
+ * switching to another browser tab after the current tab was active for at least 15 seconds (can be configured see below)
+ * navigating to another page within the same tab. 
```

Comment:
@tsteur @Findus23 I'm just having tough time understanding what Matomo's heartbeat actually does. It is stated it sends requests when user is actively viewing the page, but in next sentence it is written that is executed only on window blur. So what's true? According to https://github.com/matomo-org/matomo/pull/15058 the latter, but the name `heartbeat` suggests it makes requests in some interval…